lib/registries/addon/overview/-components/overview-topbar/template.hbs

```diff
@@ -0,0 +1,100 @@
+<div data-test-topbar-states local-class='LeftActionsContainer'>
+    <RegistriesStates @registration={{@registration}} />
+</div>
+<div data-test-topbar-share-bookmark-fork
+    local-class='RightActionsContainer'
+>
+    <ResponsiveDropdown as |dd|>
+        <dd.trigger
+            data-test-forks-dropdown-button
+            data-analytics-name='Expand fork dropdown'
+            local-class='Action'
+        >
+            <FaIcon
+                local-class='Icon'
+                @icon='code-fork'
+            />
+            <BsTooltip
+                @placement='top'
+                @triggerEvents='hover'
+            >
+                {{t 'registries.overview.tooltips.fork'}}
+            </BsTooltip>
+        </dd.trigger>
+        <dd.content>
+            <div data-test-forks-dropdown-options local-class='ForksMenu'>
+                <div local-class='ForksMenu__option'>
+                    <OsfLink
+                        data-analytics-name='Go to forks page'
+                        data-test-go-to-forks-view
+                        @route='guid-registration.forks'
+                        @models={{array @registration.id}}
+                    >
+                        {{t 'registries.overview.view_forks'}}
+                    </OsfLink>
+                </div>
+                <div local-class='ForksMenu__option'>
+                    <OsfButton
+                        data-analytics-name='Fork registration'
+                        data-test-fork-registration
+                        @type='link'
+                        @disabled={{this.forkRegistration.isRunning}}
+                        @onClick={{action (perform this.forkRegistration (action dd.close))}}
+                    >
+                        {{t 'registries.overview.fork_registration'}}
+                    </OsfButton>
+                </div>
+            </div>
+        </dd.content>
+    </ResponsiveDropdown>
+
+    <OsfButton
```

Comment:
We should hide or disable the bookmark button for anonymous users. (Will update with product's preference)